klab.hub/src/main/java/org/integratedmodelling/klab/hub/license/controllers/EngineLicenseController.java

```diff
@@ -105,8 +106,8 @@ public ResponseEntity<EngineAuthenticationResponse> processCertificate(
 		} catch (NoSuchProviderException | IOException | PGPException e) {
 			throw new LicenseGenerationError("Issue in authenticating certificate.");
 		} catch (LicenseExpiredException e) {
-			emailManager.expiredLicenseEmail(request.getEmail());
-		}
+            emailManager.expiredLicenseEmail(request.getEmail());
```

Comment:
Is this an intended change? 🤔

Reply:
I think that those are the traces of a commit that added a new `catch` scope, but was changed back to the previous state by another commit. This formatting is what remains of it.